packages/multichain-account-service/src/MultichainAccountWallet.test.ts

```diff
@@ -315,13 +316,39 @@ describe('MultichainAccountWallet', () => {
         new Error('Unable to create accounts'),
       );
 
-      const consoleSpy = jest.spyOn(console, 'warn').mockImplementation();
       await expect(
         wallet.createMultichainAccountGroup(groupIndex),
       ).rejects.toThrow(
-        'Unable to create multichain account group for index: 1',
+        'Unable to create multichain account group for index: 1 with provider "Mocked Provider 0"',
       );
-      expect(consoleSpy).toHaveBeenCalled();
+    });
+
+    it('does not fail to create an account group if a non-EVM provider fails to create its account', async () => {
+      const groupIndex = 0;
+      const mockEvmAccount = MockAccountBuilder.from(MOCK_HD_ACCOUNT_1)
+        .withEntropySource(MOCK_HD_KEYRING_1.metadata.id)
+        .withGroupIndex(groupIndex)
+        .get();
+
+      const { wallet, providers } = setup({
+        accounts: [[], []],
+      });
+
+      const [evmProvider, solProvider] = providers;
+
+      const mockSolProviderError = jest
+        .fn()
+        .mockRejectedValue('Unable to create');
+      evmProvider.createAccounts.mockResolvedValueOnce([mockEvmAccount]);
+      solProvider.createAccounts.mockImplementation(mockSolProviderError);
+
+      await wallet.createMultichainAccountGroup(groupIndex);
+
+      expect(
+        await wallet.createMultichainAccountGroup(groupIndex),
+      ).toBeDefined();
+      await new Promise(process.nextTick);
+      expect(mockSolProviderError).toHaveBeenCalled();
```

Comment:
Nit: But I think we could omit the extra mock here (since the initial plan to `await` it without using `process.nextTick`).

So I think, this could also work now

```suggestion
      evmProvider.createAccounts.mockResolvedValueOnce([mockEvmAccount]);
      solProvider.createAccounts.mockRejectedValue('Unable to create');

      await wallet.createMultichainAccountGroup(groupIndex);

      expect(
        await wallet.createMultichainAccountGroup(groupIndex),
      ).toBeDefined();
      await new Promise(process.nextTick);
      expect(solProvider.createAccounts).toHaveBeenCalled();
```